mbed-events/equeue/equeue_freertos.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
This is a bit unrelated to mbed-events. These ports come from the little [equeue](https://github.com/geky/equeue) library and the lazy manner I pulled it into mbed-events.

I don't see a reason to not integrate mbed-events completely and cull the unrelated files.